drivers/SmartThings/zigbee-motion-sensor/capabilities/sensitivityAdjustment.yaml

```diff
@@ -0,0 +1,55 @@
+id: stse.sensitivityAdjustment
+version: 1
+status: proposed
+name: Sensitivity Adjustment
+ephemeral: false
+attributes:
+  sensitivityAdjustment:
+    schema:
+      type: object
+      properties:
+        value:
+          type: string
+          enum:
+            - High
+            - Medium
+            - Low
+            - Desc
+      additionalProperties: false
+      required:
+        - value
+    setter: setSensitivityAdjustment
+    enumCommands:
+      - command: High
+        value: High
+      - command: Medium
+        value: Medium
+      - command: Low
+        value: Low
+      - command: Desc
+        value: Desc
```

Comment:
for my own education, what does `Desc` mean in this context?

Reply:
The sensitivityAdjustment capability has been extended to display not only the title but also the description to the user.